### DISCUSSION TITLE
Έκτακτη συνάντηση στα πλαίσια της ΟΣΣ για την ΓΕ2

### DISCUSSION THREAD

**Discussion by afoscolou** [sdy60/syllabus#38] [Q&A] [answered]
Κύριε Χωριανόπουλε,

Θα μπορούσαμε να οργανώσουμε μια έκτακτη συνάντηση το επόμενο Σαββατοκύριακο, 26 ή 27 Νοεμβρίου, για να συζητήσουμε τα ζητούμενα της 2ης εργασίας γιατί ήδη έχουν γεννηθεί κάποιες απορίες; Μπορεί να γίνει στα πλαίσια της ΟΣΣ2.

Σας ευχαριστούμε.

**Comment by epidrome** [comment, marked as answer]
ευχαρίστως! #39
